### PR TITLE
Bitrise script

### PR DESCRIPTION
**Pull request by felire**
## Summary

Hi people, i implemented a bitrise script to configure the project in bitrise automatically. The script will do:

- The script will create the bitrise application for the project
- The script will install a ssh key in the git project (github, gitlab or bitbucket)
- The script will install the bitrise webhook in the git repo to run the ci.

I've test the script with github and gitlab and works good. Unfortunately I could't test the script with bitbucket because i could't create personal access token in the website, I think you have to be a premium user to do it.

It's important to say that the Access token that you have to put into the script have to be like the next structure:

Github: "token 'your personal access token'"
Gitlab: "Bearer 'your personal access token'"
BitBucket: "Bearer 'your personal access token'"

The script will say that to you in the corresponding step

### CODE REVIEW

**Comment by felire** [comment]
> There are a few files without the EOF enter

done

File: generators/app/tasks/createBitriseApp.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      })
+}
+
+const createSshOnBitbucket = async (repoSlug, publicSshKey) => {
+    const userResponse = await gitApi.bitbucket.get('/user/keys');
+    const user = userResponse.data.username;
+    return await gitApi.bitbucket.post(`/users/${user}/ssh-keys`, {
+        label: `bitrise-${repoSlug}`,
+        key: publicSshKey
+    })
```
Comment (guidoprinc) [inline generators/app/tasks/createBitriseApp.js]: `;`?

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,1593 @@
+{
```
Comment (WFolini) [inline package-lock.json]: Don't commit this file. Let's use only `yarn`

File: generators/app/tasks/bitriseInitialization.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+                              message: "Write the repo slug (The name of your repo not the url)",
+                              validate: val => (val ? true : 'Repository slug is required to configure bitrise')
+                            }
+                          ]).then(({ repoSlug }) => {
+                            this.repoSlug = repoSlug;
+                            return this.prompt([
+                                {
+                                  type: 'input',
+                                  name: 'gitOwner',
+                                  message: "Who is the owner of the repo?",
```
Comment (WFolini) [inline generators/app/tasks/bitriseInitialization.js]: This is really required? 🤔

Comment (felire) [reply generators/app/tasks/bitriseInitialization.js]: Bitrise api wants that info. I suppose that it use it to send some stuff to the git api by itself

**Pull request merged**